src/modules/layout/containers/LocationLayout.js

```diff
@@ -64,24 +72,65 @@ export class LocationLayout extends React.Component<PropsType, StateType> {
       }
     }
 
+    if (location.type === EVENTS_ROUTE && events) {
+      const event = events.find(event => event.id === payload.eventId)
+      if (event) {
+        id = event.id
+        title = event.title
+      }
+    }
+
+    if (location.type === SPRUNGBRETT_ROUTE && extras) {
+      const extra = extras.find(extra => extra.alias === SPRUNGBRETT_EXTRA)
+      if (extra) {
+        alias = extra.alias
+        title = extra.title
+      }
+    }
+
+    if (location.type === WOHNEN_ROUTE && extras) {
+      const extra = extras.find(extra => extra.alias === WOHNEN_EXTRA)
+      if (extra) {
+        alias = extra.alias
+        title = extra.title
+      }
+    }
```

Comment:
Also: In each of these `if`-clauses:
What should happen, if `category` / `event` / `extra` is not found?
I think we should render some kind of error in that case

Reply:
then we should throw an error, so that we at least have some point to refer to, if the code doesn't behave as expected.

Reply:
I don't agree. The only thing this does here is that the FeedbackModal is only rendered if there is a category/event/extra. If not, no modal is rendered, which is perfectly okay, because there should be an error displayed by a Page or the Switcher anyway.

Reply:
Where should we handle this error?

Reply:
@Schedulaar ?

Reply:
Okay, np.